Cargo.toml

```diff
@@ -18,10 +18,10 @@ name = "sccache"
 name = "sccache-dist"
 required-features = ["dist-server"]
 
-[profile.release]
-codegen-units = 1
-lto = true
-strip = true
+# [profile.release]
+# codegen-units = 1
+# lto = true
+# strip = true
```

Comment:
Looks like this file change is temporary. Revert it back, please

Reply:
Yep, sorry. It made the compilation extremely slow, but I should have reverted it.